### ISSUE TITLE
Crashed when installing 'CR Downloader'

### ISSUE THREAD

**Issue by UnattendedFlight** [UnattendedFlight/Sonders-Crunchyroll-Downloader#6]
"The installer seems to crash when I try to install to default path with the error:"
```
Unhandled Exception: System.IO.IOException: File exists.

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   at CR_Downloader.Program.Install() in j:\documents\visual studio 2015\Projects\CR Downloader\CR Downloader\Program.cs:line 267
   at CR_Downloader.Program.Main(String[] args) in j:\documents\visual studio 2015\Projects\CR Downloader\CR Downloader\Program.cs:line 38
```

**Comment by UnattendedFlight** [comment]
## Explanation
This is normal, as it is trying to move some files to a folder but the folder and or file exist. 

That means CR Downloader is already installed at that location.


## Solution
Delete the already existing manager and start the installer
or
Install the manager elsewhere.

## TODO

I will add a detection of already installed CR Downloaders by writing their locations to a file in the future. 

> I will close this issue when that is done

**Comment by UnattendedFlight** [comment]
This issue has neem resolved as of commit #9
